pkg/infra/db/sqlbuilder.go

```diff
@@ -67,19 +66,10 @@ func (sb *SQLBuilder) WriteDashboardPermissionFilter(user *user.SignedInUser, pe
 		recQry       string
 		recQryParams []interface{}
 	)
-	if !ac.IsDisabled(sb.cfg) {
-		filterRBAC := permissions.NewAccessControlDashboardPermissionFilter(user, permission, queryType, sb.features, sb.recursiveQueriesAreSupported)
-		sql, params = filterRBAC.Where()
-		recQry, recQryParams = filterRBAC.With()
-	} else {
-		sql, params = permissions.DashboardPermissionFilter{
-			OrgRole:         user.OrgRole,
-			Dialect:         sb.dialect,
-			UserId:          user.UserID,
-			OrgId:           user.OrgID,
-			PermissionLevel: permission,
-		}.Where()
-	}
+
```

Comment:
Note that the permission filter is tested [here](https://github.com/grafana/grafana/blob/main/pkg/services/sqlstore/permissions/dashboard_test.go), so we can remove the tests for the old ACLs.